app/views/users/_app_permissions.html.erb

```diff
@@ -0,0 +1,36 @@
+<table class="table table-bordered table-striped table-on-white">
+  <thead>
+    <tr class="table-header">
+      <th>Application</th>
+      <% unless user_object.api_user? %>
+          <th>Has access?</th>
+      <% end %>
+      <th><%= "Other" unless user_object.api_user? %> Permissions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @application_permissions.each do |application, permissions| %>
+      <% if permissions.present? %>
+        <tr>
+          <td>
+            <% if application.retired? %>
+                <del><%= application.name %></del>
+            <% else %>
+                <%= application.name %>
+            <% end %>
+          </td>
+          <% if user_object.api_user? %>
+            <td></td>
+          <% else %>
+            <td>
+              <%= user_object.has_access_to?(application) ? "Yes" : "No" %>
```

Comment:
I like using the API here as it's easier to understand and I like working with the domain model, but, that said - we've got an array of all the permissions already fetched so we could do `permissions.include? 'signin' ? 'Yes' : 'No'` here to avoid the SQL call that `has_access_to?` will run.